src/view/search/advance-filter-modal.tsx

```diff
@@ -63,13 +63,20 @@ export const AdvancedFilterModal: FC<AdvancedFilterModalProps> = ({
 	setDateEnd
 }): ReactElement => {
 	const [otherKeywords, setOtherKeywords] = useState<KeywordState>([]);
-	const [fromDate, setFromDate] = useState<any>(dateStart);
-	const [toDate, setToDate] = useState<any>(dateEnd);
+	const [tempFromDate, setTempFromDate] = useState<Date | null>(new Date(dateStart));
```

Comment:
Please rename `tempFromDate` and `tempToDate` to `currentFromDate` and `currentToDate` respectivelly.